client/app/cloud/project/compute/infrastructure/privateNetwork/addVRack/cloud-project-compute-infrastructure-privateNetwork-addVRack.html

```diff
@@ -0,0 +1,60 @@
+<div class="oui-modal oui-modal_shadow">
```

Comment:
We have quite a few helper / components to help with modals.  It is not something final since we're awaiting the component completion on http://master.ui-kit.ovh/#!/oui-angular/introduction.  However, our components are a step in the right direction since it's allow for easy replace later on.

I'll invite you too look at name-change-modal.html

Also, see showModal in controller-modal-helper.service.js.  Do what you can, it's not really your fault since we can't talk due to time zone difference and we haven't documented our things well.

Reply:
I have used ControllerHelper to open modal. Also made changes as used in name-change-modal.html